ee/clickhouse/client.py

```diff
@@ -71,6 +79,35 @@ def async_execute(query, args=None):
         connections_max=100,
     )
 
+
+    def cache_sync_execute(query, args=None):
+        rc = redis.get_client()
+        key = hashlib.md5(query + pickle.dumps(args))
+        if rc.exists(key):
+            result = pickle.loads(rc.get(key))
+            ts = result['__ts']
+            age = datetime.datetime.now() - ts
+            if age.seconds > CACHE_TTL:
```

Comment:
Fly-by comment: Wouldn't it make sense to use redis internal ttl instead? 

https://redis-py.readthedocs.io/en/stable/#redis.Redis.set, see ex argument

Reply:
Ohhhh nice. This is perfect. Thanks!